src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucion.kt

```diff
@@ -0,0 +1,16 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDistribucion(
+    val ciudad: Ciudad
+) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun getEstrella() = vendedores.maxBy { it.puntajeCertificaciones() }
+
+    fun agregarVendedor(v: Vendedor) {
+        vendedores.find { it == v }?.let {
+            throw Exception("Vendedor ya existente!")
+        }
```

Comment:
Uy, una forma un poco rebuscada de ver si existe el vendedor. Te tiro otra:

```suggestion
        check (!vendedores.contains(v)) {
            "Vendedor ya existente!"
        }
```